plugins/actions/meson.build

```diff
@@ -0,0 +1,27 @@
+libaction_vala_sources = [
+  'action-listboxrow.vala',
+  'action-page.vala',
+  'actions.vala',
+  'action.vala',
+  'enums.vala',
+  'preferences-page.vala',
+]
+
+both_libraries(
```

Comment:
Do we really need both libraries here? Plugins should only be shared libraries.

Reply:
This just mirrors what the autotools build system does currently